doxygen/pack.dxy

```diff
@@ -868,6 +868,7 @@ INPUT           =   src/General.txt \
                     src/flash_algorithms.txt \
                     src/conditions_schema.txt \
                     src/licenses_schema.txt \
+					src/changelogs_schema.txt \
```

Comment:
Indention looks like using tabs, should be spaces to be in line:
```suggestion
                    src/changelogs_schema.txt \
```

Reply:
Fixed